config/packages/framework.yaml

```
@@ -3,6 +3,9 @@ framework:
   secret: '%env(APP_SECRET)%'
   #csrf_protection: true
   http_method_override: false
+  serializer:
+    mapping:
+      paths: [ '%kernel.project_dir%/config/serialization' ]
```

yamllint (1.35.1)

[error] too many spaces inside brackets

(brackets)